### DISCUSSION TITLE
Amethyst and Shulker Armor

### DISCUSSION THREAD

**Discussion by BL00DKNIGHT** [Roadhog360/Et-Futurum-Requiem#165] [Ideas]
The idea is that Amethyst can be used on pickaxes, axes, shovels and etc to make them a bit faster or give more damage and that amethyst has it's own armor and tools, the level of tools and armor are similar to Iron but they have an special hability, that is they can slowly regenerate or they give extra damage to mobs like if an piece of the armor broken out of the armor and sting on the mob giving damage

Shulker Armor is an armor has an level of iron between diamond in resistance, the properties of the armor is that it can launch an levitation orb that will hit mobs (like shulkers do with player) or that the armor has an shulker box in the chestplate and can be open with a keybind like an backpack
if this armor really gets added and iron (diamond, gold and etc) shulkers box too it would be cool if the shulker armor also had an iron, diamond, gold and etc version but it might be too op...

and sorry for my broken english.

**Comment by Roadhog360** [comment]
Adding new features is not within the scope of Et Futurum Requiem
